app-crypt/argon2/argon2-20171227.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="A password-hashing function"
+DESCRIPTION="Argon2 is a password-hashing function that summarizes the state \
+of the art in the design of memory-hard functions and can be used to hash \
+passwords for credential storage, key derivation, or other applications."
+HOMEPAGE="https://github.com/p-h-c/phc-winner-argon2"
+COPYRIGHT="2015-2017 Daniel Dinu
+	Dmitry Khoravich
+	Jean-Philippe Aumasson
+	Samuel Neves."
```

Comment:
Can you add the remaining authors in https://github.com/P-H-C/phc-winner-argon2/tree/20171227#intellectual-property?

Reply:
```suggestion
	Samuel Neves"
```

Also, I'm pretty sure the year range of "Daniel Dinu" applies to the contributors below as well.

Reply:
Only found one missing name there, updated the year range